neo4j-opencypher-dsl/src/main/java/org/neo4j/opencypherdsl/PatternComprehension.java

```diff
@@ -63,7 +63,8 @@ static OngoingDefinitionWithPattern basedOn(RelationshipChain pattern) {
 	/**
 	 * Provides the final step of defining a pattern comprehension.
 	 */
-	public interface OngoingDefinitionWithoutReturn {
+	public interface OngoingDefinitionWithoutReturn
+		extends StatementBuilder.ExposesConditions<OngoingDefinitionWithoutReturn> {
```

Comment:
That won't work on this level, as it would expose the operator directly without the where.